src/howtoplay/howtoplay.scss

```diff
@@ -13,50 +13,63 @@
   flex-direction: column;
   justify-content: space-between;
   margin: auto;
-  @media screen and (min-width: $l-breakpoint) {
-    width: 65vw;
+  width: 80%;
+  @media screen and(orientation: landscape) {
+    width: 95%;
+  }
+  @media screen and (min-width: $m-breakpoint) {
+    max-width: 950px;
   }
 }
 .container {
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: flex-start;
-
   &__title {
     font-size: $m-font-size-mobile;
     font-weight: 700;
     padding: 0 10px 10px 10px;
     border-bottom: 5px solid $white-color;
     text-align: center;
-    margin-bottom: 5%;
-    margin-top: 5%;
+    margin: 20px 0;
     @media screen and (min-width: $m-breakpoint) {
       font-size: $l-font-size-desktop;
+      margin: 60px 0 100px 0;
```

Comment:
For mobile, the margin between instruction and header is around 50px.

Reply:
i know but our font-size is 16px instead 13px. I can't  use margin 50px

Reply:
change to 13px